### PR TITLE
[Affiliate] Copy link component

### LINKED ISSUE #851: [Affiliate] Show share buttons
**Description**
Show share buttons

**Proposal**
- Copy url: https://cowswap.exchange/#/?referral=0x123...0  (IPFS hash)

**Additional context**
Draft wireframes proposal https://www.figma.com/file/P0AIwKkIUpEZtLukkCUM94/CowSwap_-Affiliate?node-id=48%3A224
Affiliate main doc https://docs.google.com/document/d/1PsdWn9WPFf2BNeoJNtClyra8jNvWk_bJ-g33TbpUf8s/edit#heading=h.wc3ppxg7l1mo

### PR DESCRIPTION
**Pull request by matextrem**
# Summary
Closes #851 

Edit Copy component in order to allow copying a link either by clicking over the icon or address. 

As part of the discussion [here](https://github.com/gnosis/cowswap/pull/952#pullrequestreview-708958950) I decided to split this issue in 2 PRs (this is the one related to the implementation and then there will be another related to the integrations once the affiliate page is merged)

# Testing steps
Since it is not a integration and given the fact we don't have Storybook to list components, there is no visual feedback until it is integrated into the affiliate page.

### CODE REVIEW

**Review by ramirotw — CHANGES_REQUESTED** [review]
Nice!

The `Copy.tsx` file is under the `AccountDetails` folder, not sure why.

Also, I'd remove the changes in the About page as they are not related.

File: src/custom/components/AccountDetails/Copy.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import useCopyClipboard from '@src/hooks/useCopyClipboard'
```
Comment (alfetopito) [inline src/custom/components/AccountDetails/Copy.tsx:3]: We prefer to use the `@src` reference only when required.
This should work with the default path.

I know this import comes automatically from VSCode, just keep an eye out :)

```suggestion
import useCopyClipboard from 'hooks/useCopyClipboard'
```

File: src/custom/pages/About/index.tsx [hunk 2]
```diff
@@ -30,7 +30,6 @@ export default function About() {
         <h2>
           <b>C</b>oincidence <b>o</b>f <b>W</b>ants
         </h2>
-
```
Comment (alfetopito) [inline src/custom/pages/About/index.tsx:33]: Don't think this change is needed

**Review by alfetopito — CHANGES_REQUESTED** [review]
Tiny change regarding the `@src` path usage.

And I agree with Ramiro regarding the file location.
Think a best place would be `/src/custom/components/Copy/index.tsx`

**Comment by matextrem** [comment]
> Tiny change regarding the `@src` path usage.
> 
> And I agree with Ramiro regarding the file location.
> Think a best place would be `/src/custom/components/Copy/index.tsx`

I left the same location used for non custom components but I do agree is odd. I'll change it.

**Review by alfetopito — APPROVED** [review]

**Review by ramirotw — APPROVED** [review]

**Pull request merged**